addons/l10n_it_edi/views/l10n_it_view.xml

```diff
@@ -26,7 +26,7 @@
                     <field name="l10n_it_vat_due_date"/>
                     <field name="l10n_it_has_exoneration" readonly="False"/>
                     <field name="l10n_it_kind_exoneration" attrs="{'invisible': [('l10n_it_has_exoneration', '=', False)]}"/>
-                    <field name="l10n_it_law_reference" attrs="{'invisible': [('l10n_it_has_exoneration', '=', False)]}"/>
+                    <field name="l10n_it_law_reference"/>
```

Comment:
@lordkrandel Do not we need to change in https://github.com/odoo/odoo/blob/1b8b1633cc04d11343ee69f4b687c8fd35ab460a/addons/l10n_it_edi/data/invoice_it_template.xml#L174 as well?

Reply:
Oh! Yes of course.